server/etc/pulp/server.conf

```diff
@@ -164,7 +163,6 @@
 
 [data_reaping]
 # reaper_interval: 0.25
-# archived_calls: 0.5
```

Comment:
I disagree - I think we can remove this setting since it is getting removed elsewhere. It doesn't make sense to mark it deprecated when it actually doesn't do anything. Also, users who have this setting present in their config file will not experience any problems - it just doesn't get used.

I do think we should mention in the release notes that this setting is not used anymore.